### ISSUE TITLE
[Feature]: Option to change the song (list)

### ISSUE THREAD

**Issue by NightFly88** [fast4x/RiMusic#1325]
### Description

I cannot find in favourites and playlists an option to change the song 
![Screenshot_20240331_213814_YouTube Music](https://github.com/fast4x/RiMusic/assets/163666747/b81db0ea-9eb4-42d6-8f83-0867b883465b)
(list),youtube music does have this option,could this be added

Thanks

### Suggested Solution

I would like to see this option,

**Comment by fast4x** [comment]
Re arrange song is available in playlists when sorting for position
![Screenshot_2024-04-01-01-29-57-998_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/1484476/01c47932-c043-473d-ad45-4bedd5cd3bbd)

**Comment by sayyid5416** [comment]
I think he meant in `Favorites` playlist.

**Comment by fast4x** [comment]
he has write: favourites and playlists...

**Comment by NightFly88** [comment]
I would like to sort the songs myself

**Comment by NightFly88** [comment]
> he has write: favourites and playlists...

Yes

**Comment by fast4x** [comment]
it's possible only in playlists...

**Comment by NightFly88** [comment]
https://github.com/fast4x/RiMusic/assets/163666747/0b830cb6-4e1a-41b4-ab0f-d9c337e25e30

I mean this see youtube music

**Comment by NightFly88** [comment]
> it's possible only in playlists...


https://github.com/fast4x/RiMusic/assets/163666747/32ab2f3f-d198-4c95-a515-6955d5633e78

See video YouTube Music

**Comment by fast4x** [comment]
Yes I've understand, it's possible only in playlists.https://github.com/fast4x/RiMusic/assets/1484476/ef18745c-6dc6-4c00-ae07-6213571f344a

Sort for position, unlock reorder and that's all

**Comment by NightFly88** [comment]
> Yes I've understand, it's possible only in playlists.https://github.com/fast4x/RiMusic/assets/1484476/ef18745c-6dc6-4c00-ae07-6213571f344a
> 
> Sort for position, unlock reorder and that's all

Yes its works many thanks
